src/corg/development-tools-and-process/quality-assurance-and-continuous-integration/isolated-testing.md

```diff
@@ -151,36 +151,6 @@ You need to use the `customdb` profile to init build environment, and then desir
 
 You also need to use a slave owning the configuration for desired DB, for example `MULTIDB_LINUX`.
 
-### Build Environment
-
-### Discard Old Build
-{{#> panel type='code' heading='JenkinsFile'}}
-
-```
-properties([[$class: 'BuildDiscarderProperty',                                                             
-                     strategy: [$class: 'LogRotator', artifactDaysToKeepStr: '',                           
-                     artifactNumToKeepStr: '1',                                                            
-                     daysToKeepStr: '60',                                                                  
-                     numToKeepStr: '60']]])
-```
-{{/panel}}
-
-#### Abort the Build If It's Stuck + Add Timestamps to the Console Output
-{{#> panel type='code' heading='Jenkinsfile'}}
-```
-timestamps {
-        timeout(time: 240, unit: 'MINUTES') {
-                   // code here
-                }
-           }
- ```
-{{/panel}}
-
-### Build
-
-#### Execute Shell
-
-{{#> panel type='code' heading='Shell script'}}
```

Comment:
Hi Alexis, you remove a panel here but you forgot to remove the closing tag. Could you please fix it?